conferences/SC21/bof_draft.md

```diff
@@ -0,0 +1,115 @@
+## Title: HPC Carpentry: Introducing New Users to HPC
+
+## Session Leaders
+
+ - Andrew Reid
+ - Trevor Keller
+ - Others?
+
+It's possible to include a brief bio for these.
+
+## Topic Area
+
+ - Education
+
+## Abstract:
+
+[HPC Carpentry][1] is a community and a set of lessons for introducing new
+users to cluster-based high-performance computing resources. As these resources
+become more common, the need for educational resources to bring users from a
+broad variety of technical backgrounds up to speed becomes more urgent. Based
+on the pedagogically sound methods of the larger [Carpentries][2] method,
+lessons in active development include an [introduction][3] to the use of a
+queuing system, an introduction to [parallel programming in Python][4], and a
+lesson focusing on the [Chapel parallel language][5]. The introductory lesson
+is part of the Carpentries incubator program, and is on track to be
+incorporated in to the Carpentries as a regular lesson in the near future.
+
+  
+##  Long description:
+
+### Who we are
+
+The [HPC Carpentry][1] community, a volunteer group of high-performance
+computing practitioners lead by a [steering committee][6], has developed a
+number of introductory lessons for new users of HPC clusters.
+
+### What Problem We Solve
+
+It is almost a cliche at this point that greater and greater computational
+capabilities are becoming increasingly affordable. An under-appreciated
+consequence of this is that the range of backgrounds of new users of these
+resources is increasingly broad, and may not include knowledge of computer
+system configurations and file system layouts. There is an evident on-going
+need to address the gap between the needs of these users and the capabilities
+and opportunities presented by the available HPC resources.
+
+The [HPC Carpentry][1] lessons address this gap, building on the pedagogically
+sound instructional methods of the larger [Carpentries][2] community to develop
+several lessons. The best-developed of these, the [HPC Intro][3] lesson, was
+recently submitted to the [Carpentries Incubator][7] program, and is on track
+to become an official [Carpentries][2] lesson. Other important lessons in
+development include a [parallel python][4] lesson, and a lesson focusing on the
+[chapel][5] language.
```

Comment:
What about the HPC Shell lesson, and I'm not sure I would highlight Chapel unless there is a serious plan to update that.

Reply:
Also, in general, how do we acknowledge that other people are also creating lessons that are potentially of huge value to HPC Carpentry but are not under that umbrella (the [container lesson from the Pawsey Centre](https://pawseysc.github.io/singularity-containers/) is a good example)